tailwind.common.js

```diff
@@ -50,6 +51,7 @@ module.exports = {
         200: 'rgb(var(--purple-200) / <alpha-value>)',
         100: 'rgb(var(--purple-100) / <alpha-value>)',
       },
+      white: 'rgb(var(--white) / <alpha-value>)',
```

Comment:
Ok, I didn't see this. Why call a CSS variable with the value white (255 255 255) --grey-0? Same for --grey-700, it probably should be called black. Naming should be descriptive of the color they represent to avoid confusion.

Reply:
this is the grey color scale names used on figma so we just kept it for dev as well:

![image](https://github.com/user-attachments/assets/b0be3f57-f921-4c2e-8e7e-4ef1f95e3ddf)

Also see https://www.figma.com/design/Md1Cl2B9eJi4NMzQgpbdMF/Dark-Mode?node-id=0-1&t=oX5d1JZj22LQ6RGi-0
We didn't add the actual color name, ie black, because in dark mode black is actually white, as you can see in our tailwind.css. And it would make the color names longer to write down.

Reply:
Maybe we should use a different approach then. For example, something like:

 ```
 :root {
    --main-text-color: var(--grey-700); // or --black
  }

  .dark {
    --main-text-color: var(--grey-0); // or --white
  }
```

I understand this is out of scope, so I'll update the code as per your first comment.